src/components/list/PagedList.js

```diff
@@ -0,0 +1,181 @@
+import { useRect } from '@reach/rect'
+import { ChevronLeft, ChevronRight } from '@styled-icons/boxicons-regular'
+import { useEffect, useRef, useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+import { useMap } from '../../contexts/MapContext'
+import { getLocations } from '../../utils/api'
+import Checkbox from '../ui/Checkbox'
+import LoadingIndicator from '../ui/LoadingIndicator'
+import SquareButton from '../ui/SquareButton'
+import EntryList from './EntryList'
+
+const LIMIT = 30
+
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`
+
+const StyledListContainer = styled.div`
+  height: 100%;
+`
+
+const StyledPageInfo = styled.div`
+  display: flex;
+  flex-direction: column;
+  visibility: ${({ visible }) => (visible ? 'visible' : 'hidden')};
+`
+
+const StyledPageNav = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+  border-top: 1px solid ${({ theme }) => theme.secondaryBackground};
+`
+
+const StyledNavButtonContainer = styled.div`
```

Comment:
You can omit `Styled` prefix from components that don't need it (if there isn't already a component that's named that).